BooruSharp/Booru/SankakuComplex.cs

```diff
@@ -1,10 +1,18 @@
 ﻿namespace BooruSharp.Booru
 {
+    /// <summary>
+    /// Sankaku Complex.
+    /// <para>https://chan.sankakucomplex.com/</para>
```

Comment:
We can probably redirect to the new website instead: https://beta.sankakucomplex.com/posts/

Reply:
I think it's better left as is, since the new website layout is in beta.

Reply:
It's been in beta since 2018 so I'm not sure we can still consider this a "beta" (especially since the API we are using in the one of the beta website too)